utils.py

```diff
@@ -107,6 +107,8 @@ def publish(temperatures, battery, heating_element, client, base_topic, device_n
         client.publish("{0}/{1}/battery".format(base_topic, device_name), battery)
     if heating_element:
         client.publish("{0}/{1}/heating_element".format(base_topic, device_name), heating_element)
+    if propane_level:
```

Comment:
If propane_level is `0.0` the if test will fail, and no level will be transmitted.

Reply:
```suggestion
    if propane_level is not None:
```

Reply:
I think that should work. Started a new job recently and just haven't had a chance to go back and test, but I believe you're right that that is the change that needs to be made @eth0up.